payer rules (#112)
Mangopay/docs: guides/users/categories/restrictions.mdx

```
</Accordion>

## Transfers can't be credited to wallets owned by Payers

Payers are not allowed to receive transfers except when:

> 1. the credited and debited wallets have the same owner

Reviewer: ```suggestion
1. the credited and debited wallets belong to the same owner
```
I think it sounds clearer, explicitly draws out the relationship of users to wallets

> 2. the ttransfer is debited from user with a category: Platform.

Reviewer: ```suggestion
2. the transfer is debited from user with a category: Platform.
```

> Otherwise a request to “Create a Transfer“ endpoint will fail with response code `002701`.

Reviewer: ```suggestion
Otherwise a request to “Create a Transfer“ endpoint will result in a creation of a failed transfer with response code `002701`.
```
Technically the call will succeed, but the created transfer will have its result/error code set to 002701.

## Payouts can't be debited from wallets owners by Payers

> Payouts can't be debited from wallets owners by Payers except when `PaymentRef` parameter is provided on the [POST Create a Payout](/api-reference/payouts/create-payout) endpoint with `ReasonType` to `PAYIN_REFUND` and `Id` of the initial pay-in as the `ReferenceId`.

Reviewer: ```suggestion
Payouts can't be debited from wallet owners by Payers except when `PaymentRef` parameter is provided on the [POST Create a Payout](/api-reference/payouts/create-payout) endpoint with `ReasonType` to `PAYIN_REFUND` and `Id` of the initial pay-in as the `ReferenceId`.
```

> Otherwise a request to “Create a Payout” endpoint will fail with a response code `002701`.

Reviewer: ```suggestion
Otherwise a request to “Create a Payout” endpoint will result in a creation of a failed payout with a response code `002701`.
```

## KYC Documents and UBO declarations can't be submitted for Payers

> Payers are not able to request KYC [verification](/guides/users/verification) (by submitting one or more [KYC Documents](/api-reference/kyc-documents/kyc-document-object)). Otherwise, a request to "Submit a KYC Document" will results in `KYC_FAILED` event.

Reviewer: ```suggestion
Payers are not able to request KYC [verification](/guides/users/verification) (by submitting one or more [KYC Documents](/api-reference/kyc-documents/kyc-document-object)). Otherwise, a request to "Submit a KYC Document" will result in creation of a rejected document (with Status property set to REFUSED) and a `KYC_FAILED` event will be issued.
```

> Payers are not able to declare Ultimate Beneficial Owners in a [UBO Declaration](/api-reference/ubo-declarations/ubo-declaration-object). Otherwise, a request to "Create a UBO Declaration" wil return an error:

Reviewer: ```suggestion
Payers are not able to declare Ultimate Beneficial Owners in a [UBO Declaration](/api-reference/ubo-declarations/ubo-declaration-object). Should such request be made to "Create a UBO Declaration" endpoint, it will return an error:
```
